Workflow for deploying to openscd.github.io is broken

**Describe the bug**
Our "Build and Deploy" workflow currently fails at running the `doc` at the core package because of what it seems to be conflicting versions of devDependencies: typedoc and typescript with sibling packages open-scd and plugins.

Here's the latest output we have for that github action:
```terminal
> @openscd/core@0.1.1 doc
> typedoc --out doc foundation.ts

error The plugin /home/runner/work/open-scd/open-scd/node_modules/typedoc-plugin-markdown could not be loaded.
error Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './dist/lib/converter/plugins' is not defined by "exports" in /home/runner/work/open-scd/open-scd/node_modules/typedoc/package.json
    at new NodeError (node:internal/errors:405:5)
    at exportsNotFound (node:internal/modules/esm/resolve:366:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:713:9)
    at resolveExports (node:internal/modules/cjs/loader:590:36)
    at Module._findPath (node:internal/modules/cjs/loader:664:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1126:27)
    at Module._load (node:internal/modules/cjs/loader:981:27)
    at Module.require (node:internal/modules/cjs/loader:1231:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/home/runner/work/open-scd/open-scd/node_modules/typedoc-plugin-markdown/dist/theme.js:13:19)
npm ERR! Lifecycle script `doc` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: @openscd/core@0.1.1 
npm ERR!   at location: /home/runner/work/open-scd/open-scd/packages/core 
```

**To Reproduce**
Steps to reproduce the behavior:
1. Merge a PR into main
2. Workflow "Build and deploy" fails

**Expected behavior**
"Build and deploy" workflow works and all the packages generate their respective typedoc

--- a/packages/addons/package.json
+++ b/packages/addons/package.json
@@ -23,7 +23,7 @@
     "@openscd/core": "*",
     "@openscd/components": "*"
   },
-  "scripts": {
+  "//": {
     "clean": "rimraf build",
     "lint:eslint": "eslint --ext .ts . --ignore-path .gitignore",
     "format:eslint": "eslint --ext .ts . --fix --ignore-path .gitignore",

--- a/packages/components/package.json
+++ b/packages/components/package.json
@@ -20,7 +20,7 @@
   ],
   "dependencies": {
   },
-  "scripts": {
+  "//": {
     "clean": "rimraf build",
     "lint:eslint": "eslint --ext .ts . --ignore-path .gitignore",
     "format:eslint": "eslint --ext .ts . --fix --ignore-path .gitignore",

--- a/packages/distribution/package.json
+++ b/packages/distribution/package.json
@@ -24,7 +24,7 @@
     "@openscd/addons": "*",
     "@openscd/plugins": "*"
   },
-  "scripts": {
+  "//": {
     "clean": "rimraf build",
     "lint:eslint": "eslint --ext .ts . --ignore-path .gitignore",
     "format:eslint": "eslint --ext .ts . --fix --ignore-path .gitignore",

--- a/packages/wizards/package.json
+++ b/packages/wizards/package.json
@@ -23,7 +23,7 @@
     "@openscd/core": "*",
     "@openscd/components": "*"
   },
-  "scripts": {
+  "//": {
     "clean": "rimraf build",
     "lint:eslint": "eslint --ext .ts . --ignore-path .gitignore",
     "format:eslint": "eslint --ext .ts . --fix --ignore-path .gitignore",

